experimental/auth-logic/unix_epoch_time_wrapper_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package authlogic contains logic and tests for interfacing with the
+// authorization logic compiler
+package authlogic
+
+import (
+	"io/ioutil"
+	"regexp"
+	"strconv"
+	"testing"
+)
+
+func (time UnixEpochTime) Identify() Principal {
```

Comment:
```suggestion
func (time UnixEpochTime) Identify() principal {
```
I assume the name does not need to be exported.

Reply:
This is actually the return type defined in another file. It does need to be exported from the package so that other packages can implement the `Wrapper` interface if desired.

Reply:
Everything is in the `authlogic` package. Right? I thought they'd be accessible in all these files, even without being exported.